contracts/libraries/PostLib.sol

```diff
@@ -946,7 +946,7 @@ library PostLib  {
         self.peeranhaUser.updateUsersRating(usersRating, communityId);
     }
 
-    // @notice Recalculation rating for all users who were active in the post
+    // @notice Recalculation rating for all users who were active in the post when postType has been changed
     /// @param self The mapping containing all posts
```

Comment:
How about we move `postContainer.info.postType = postType;` inside this function and rename the function to `changePostType`? Function name should mean an action.